search/search-domain/src/main/java/io/camunda/search/filter/ProcessInstanceFilter.java

```diff
@@ -51,37 +50,34 @@ public static final class Builder implements ObjectBuilder<ProcessInstanceFilter
     private Boolean hasIncident;
     private List<Operation<String>> tenantIdOperations;
 
-    @SafeVarargs
-    private <T> List<Operation<T>> mapToOperationEq(final T... values) {
-      return Arrays.stream(values).map(Operation::eq).collect(Collectors.toList());
-    }
-
     public Builder processInstanceKeyOperations(final List<Operation<Long>> operations) {
       processInstanceKeyOperations = addValuesToList(processInstanceKeyOperations, operations);
       return this;
     }
 
-    public Builder processInstanceKeys(final Long... values) {
-      return processInstanceKeyOperations(mapToOperationEq(values));
+    public Builder processInstanceKeys(final Long value, final Long... values) {
```

Comment:
❓ What is the benefit of using this method signature compared to before?

Reply:
This way it needs to be called with at least one parameter. It can't be used like `processInstanceKeys()`, which wouldn't make sense.

Reply:
I see, yeah, that makes sense, compile-time clarity on how you can use the API. Sure 👍